blogapp/app/views/posts/show.html.erb

```diff
@@ -18,3 +18,4 @@
 </p>
```

Comment:
- [ ] I can not see any button or anything to click to like a post or comment on a post, kindly fix that
![Screenshot from 2023-07-28 19-09-58](https://github.com/OnlyUnknown/blog-app-v3/assets/66097371/9ce8aac0-6e90-49f2-8c5c-c16f2ffb254e)